changelogs/unreleased/gh-9924-luajit-fixes.md

```diff
@@ -0,0 +1,17 @@
+## bugfix/luajit
+
+Backported patches from the vanilla LuaJIT trunk (gh-9924). The following issues
+were fixed as part of this activity:
+
```

Comment:
I would also mention added UBSan support and seems commit "Prevent sanitizer warning in snap_restoredata()." is missing in changelog.

Reply:
The sanitizer warning in `snap_restoredata()` is also harmless since it can't lead to any misbehaviour, so it is not visible to the user, so it isn't mentioned in the changelog.

Reply:
> snap_restoredata

ok

what about added ubsan support to changelog?

Reply:
In a private conversation decided to not add ubsan entry to changelog.

Reply:
IMHO, it is not a major thing since Tarantool itself has UBSan support. LuaJIT may not be so interesting for users after all.